inc/package.class.php

```diff
@@ -190,7 +190,7 @@ public function prepareInputForAdd($input) {
       }
       try {
          $uploadedFile = $preparedFile['uploadedFile'];
-         $input['filename'] = 'flyvemdm/package/' . $this->fields['entities_id'] . '/' . uniqid() . '_' . basename($uploadedFile);
+         $input['filename'] = 'flyvemdm/package/' . $input['entities_id'] . '/' . uniqid() . '_' . basename($uploadedFile);
```

Comment:
$this->fields should not contain anything or is garbage if an instance is r e-used.